### ISSUE TITLE
Errors due to a git pulling

### ISSUE THREAD

**Issue by bserma** [cisco/libacvp#355]
Hello,

Recently, I update my depot with a git pull and when I try to recompile with the command "make",  I have this error message.
`acvp.c:24:22: fatal error: safe_lib.h`

It worked before and now it does not work.
What is the source of my problem ?

I try to override the problem in using the docker of libacvp but some problems appear:
with the openssl library 102 -> `app_main.c:18:10: fatal error: openssl/rsa.h: not found`
with the openssl library 110 -> the docker does not build.
with the openssl library 111 -> `ACVP [ERR][acvp_curl_http_post:686]--> Curl failed with code 60`

So to summarize:
I had one problem, I tried to resolve with 3 possible solutions but now I have 3 problems ^^
Can you help me please ?

Best regards,

**Comment by bfussell** [comment]
First, I don't think our intention was to continue to support openssl 1.1.0, that verion of openssl was EOL some time ago and was never a long term support branch.  We do support openssl 1.1.1 and 1.0.2.  

Your first error looks like a path error such that your build cannot find a header file which should be included from ./safe_c_stub/ and your compile options should include a -I<path to ./safe_c_stub/>

I just built 1.0.2 with no issues using the docker ./configure command.
Your 1.1.1 curl error was discussed on an earlier issue #126 that may be of help.

**Comment by bserma** [comment]
Hello,

Thank you for your response.
I understood my mistake and now the docker works with openssl 1.0.2 and the CA certificate mozzila_trust_anchors.pem but it does not work with acvp.nist.gov.crt. 

I don't understand the error `Curl failed with code 60`. This error is due to SSL peer certificate but to my mind the "Nist" certificate should work.

Best regards
